packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/ClusterManagersController.m

```diff
@@ -0,0 +1,157 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "ClusterManagersController.h"
+#import "FLTGoogleMapJSONConversions.h"
+
+@interface FLTClusterManagersController ()
+
+@property(strong, nonatomic) NSMutableDictionary *clusterManagerIdToManagers;
+@property(strong, nonatomic) FlutterMethodChannel *methodChannel;
+@property(weak, nonatomic) GMSMapView *mapView;
+
+@end
+
+@implementation FLTClusterManagersController
+- (instancetype)init:(FlutterMethodChannel *)methodChannel mapView:(GMSMapView *)mapView {
+  self = [super init];
+  if (self) {
+    _methodChannel = methodChannel;
+    _mapView = mapView;
+    _clusterManagerIdToManagers = [[NSMutableDictionary alloc] init];
+  }
+  return self;
+}
+
+- (void)addClusterManagers:(NSArray *)clusterManagersToAdd {
+  for (NSDictionary *clusterManager in clusterManagersToAdd) {
+    NSString *identifier = clusterManager[@"clusterManagerId"];
+    id<GMUClusterAlgorithm> algorithm = [[GMUNonHierarchicalDistanceBasedAlgorithm alloc] init];
+    id<GMUClusterIconGenerator> iconGenerator = [[GMUDefaultClusterIconGenerator alloc] init];
+    id<GMUClusterRenderer> renderer =
+        [[GMUDefaultClusterRenderer alloc] initWithMapView:_mapView
+                                      clusterIconGenerator:iconGenerator];
+    GMUClusterManager *clusterManager = [[GMUClusterManager alloc] initWithMap:_mapView
```

Comment:
nits: s/_mapView/self.mapView

Reply:
Can you explain little bit more how this should be changed?